### PR TITLE
Add Studydrive GmbH

### PR DESCRIPTION
**Pull request by zner0L**

### CODE REVIEW

File: companies/studydrive.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "slug": "studydrive",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Studydrive GmbH",
+    "address": "Mehringdamm 32\n10961 Berlin\nDeutschland",
+    "phone": "+49 (030) 555 794 64",
```
Comment (baltpeter) [inline companies/studydrive.json]: If I type that into the JSON generator, I (correctly) get the error that it doesn't match the format regex.

File: companies/studydrive.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "email": "info@studydrive.net",
+    "web": "https://www.studydrive.net",
+    "sources": [
+        "https://www.studydrive.net/datenschutz",
+        "https://www.studydrive.net/impressum"
+    ],
+    "required-elements": [
```
Comment (baltpeter) [inline companies/studydrive.json]: We had agreed to always include the name element.

*You* actually insisted on that. :P

File: companies/studydrive.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        {
+            "desc": "E-Mail-Adresse",
+            "type": "input",
+            "optional": false
+        }
+    ],
+    "suggested-transport-medium": "email"
```
Comment (baltpeter) [inline companies/studydrive.json]: Why is the stm email when we only have the generic `info@`.

**Comment by zner0L** [comment]
> Oh yes, I definitely need to request my data from them!

That's why I'm adding it.

File: companies/studydrive.json (hunk 3 above)
Comment (zner0L) [reply companies/studydrive.json]: They explicitly ask for requests to be sent to this email.

File: companies/studydrive.json (hunk 2 above)
Comment (zner0L) [reply companies/studydrive.json]: Well…

File: companies/studydrive.json (hunk 3 above)
Comment (baltpeter) [reply companies/studydrive.json]: Sure, but our current policy is to only suggest email if the email explicitly goes to the DPO or privacy department or whatever.

File: companies/studydrive.json (hunk 1 above)
Comment (zner0L) [reply companies/studydrive.json]: WTF? This is different from what was in the input. Also, the format button in the JSON generator seems to be broken for me. I don't know the expected behavior, but it does definitely not format…

File: companies/studydrive.json (hunk 3 above)
Comment (zner0L) [reply companies/studydrive.json]: *in their privacy policy

Comment (baltpeter) [reply companies/studydrive.json]: I know, I looked at it.

Comment (zner0L) [reply companies/studydrive.json]: Ok, sure.

File: companies/studydrive.json (hunk 1 above)
Comment (baltpeter) [reply companies/studydrive.json]: As I said, there are some (a lot of) problems with the JSON forms thingy.

Peek 2019-10-05 20-00

The format button (pressing enter in the input also works) works fine for me. The problem is that JSON forms doesn't seem to recognize that the input has changed. You need to manually change it, trigger a read and then change it back. Very annoying.

Similar issues occur when dragging and dropping text into the inputs for example. C&P-ing usually works, though.

**Pull request merged**